src/utils/backend.js

```diff
@@ -45,6 +45,21 @@ export default class Backend {
   getProfileImage = async (address) => wrapTry(async () => {
     return fetch(`${this.BACKEND_URL}/profile/image/${address}`);
   });
+  setProfileImage = async (address, data, image = true) => wrapTry(async () => {
+    if (image) {
+      return fetch(`${this.BACKEND_URL}/profile/image/` + address, {
+        method: 'post',
+        body: data,
+      });
+    }
+    else {
```

Comment:
else is uneeded

Reply:
It is needed as we send different `Content-Type` on the follow-up request. The avatar upload is 2 step process: uplaod image -> get challenge -> sign and respond.

Reply:
i mean, return will exit the function else will not be reached in this case.

Reply:
`image` flag is used to check what request are you sending to the backend .. if its `true` it gets in the first condition, otherwise it gets in the `else`.

Reply:
I mean this is essentially the same
```
if (image) {
      return fetch(`${this.BACKEND_URL}/profile/image/` + address, {
        method: 'post',
        body: data,
      });
    }

      return fetch(`${this.BACKEND_URL}/profile/image/` + address, {
        method: 'post',
        body: JSON.stringify(data),
        headers: {'Content-Type': 'application/json'}
      });
```

Reply:
![image](https://user-images.githubusercontent.com/632282/77146050-c6b8f100-6a92-11ea-8c50-2d7b6b0716fa.png)

Reply:
I've simplified it in https://github.com/aeternity/aepp-raendom/pull/183/commits/61eb92f9059ae6a6ca564de11ed49608a8ba7df5